browser/scripts/node-ui.js

```diff
@@ -270,41 +338,37 @@ NodeUI.prototype.canDisplayInputInHeader = function() {
 NodeUI.prototype.canDisplayOutputInHeader = function() {
 	var p = this.getPluginUIFlags();
 	var can = p.single_out && (!p.has_edit) && (!p.has_dynamic_slots);	// check !p.has_inputs if stricter
-	can &= !p.has_subgraph;
+	can = can && !p.has_subgraph;
 
 	var exceptPlugins = ['envelope_modulator'];
 
 	var myCategory = this.getNodeCategory();
-	can &= (uiNodeCategoriesThatNormallyDisplayOutputInHeader.indexOf(myCategory) > -1);
-	can &= (exceptPlugins.indexOf(this.parent_node.plugin.id) === -1);
+	can = can && (uiNodeCategoriesThatNormallyDisplayOutputInHeader.indexOf(myCategory) > -1);
+	can = can && (exceptPlugins.indexOf(this.parent_node.plugin.id) === -1);
 
 	return can;
 };
 
 NodeUI.prototype.canDisplayInline = function() {
```

Comment:
you were right about those &='s too. jshint caught them.
i'll do a unit test once we have all designs.